### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by shrektan**

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Thanks

Next time you had TWO at the same time consider amalgamating them into the same PR.

**Comment by eddelbuettel** [comment]
Oh, you had closed one already.  Now I understand even of what you did.

Thanks anyway.

File: README.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ Starting with release 0.4.3, binaries are provided via the [ghrr drat repo](http
 and can be installed as
 
 ```{r}
-R> drat::addRepo("ghrr")       #  maybe use 'install.package("drat")' first 
-R> install.packages(RQuantLib, type="binary")
+R> drat::addRepo("ghrr")       #  maybe use 'install.packages("drat")' first 
+R> install.packages("RQuantLib", type="binary")
```
Comment (eddelbuettel) [inline README.md]: The quotes are not needed, and I usually do not place them.

**Comment by eddelbuettel** [comment]
I used to opportunity to clean this up ever so slightly with one more commit.  Thanks again.

File: README.md (hunk 1 above)
Comment (shrektan) [inline README.md]: Actually, I think the quotes are needed...

```r
install.packages(RQuantLib, type="binary")
## Error in install.packages : object 'RQuantLib' not found
```

**Comment by shrektan** [comment]
You're welcome. Thanks for packages that you contributed. Powerful and elegant! :smile:

**Comment by eddelbuettel** [comment]
:-)

**Pull request merged**